locales/pt-BR/translation.json

```
@@ -90,6 +90,29 @@
         "workday-waiver-manager": "Gerenciador de Folgas",
         "yes-please": "Sim, por favor"
     },
+    "$FixedCalendar": {
+        "leave-by": "Baseado na hora que chegou hoje, você deve sair às"
+    },
+    "$FlexibleDayCalendar": {
+        "switch-view": "Trocar Visualização",
+        "current-day": "Ir para o Dia Atual",
+        "previous-day": "Dia Anterior",
+        "time-to-leave": "Hora de Ir Embora",
+        "leave-by": "Você deve sair às:",
+        "day-done-balance": "Tudo finalizado por hoje. Saldo do dia:",
+        "balance-up-until-today-for-this-month": "Saldo até'hoje para este mês. Um saldo positivo significa horas extras que você não precisa trabalhar hoje (ou o resto do mês)",
+        "month-balance": "Saldo Mensal",
+        "overall-balance-month": "Saldo total até o fim do mês ou dia atual",
+        "overall-balance": "Saldo Total",
+        "not-a-working-day": "Nao é um dia útil",
+        "waived-day": "Dia dispensado",
```

We have been using `folga`, for `waived` translation:

```suggestion
        "waived-day": "Dia de folga",
```